doc/source/param/method_check.incl

```diff
@@ -0,0 +1,81 @@
+The ``"check"`` method is used for writing HDF5 data files used by
+Cello for restart. The ``"check"`` method can also be used for data
+dumps, though currently all data (all fields and all particles on all
+blocks) are written to disk.  This method should typically be called
+with a schedule.
+
+**Warning 1: Currently requires the** ``"order_morton"`` **method to be called beforehand, with a matching schedule.**
+
+**Warning 2: If a file or directory corresponding to a requested checkpoint
+dump already exists, no data will be written.** This is to avoid over-writing
+checkpoint files, but may not necessarily be what is expected. This behavior
+may be revised in the future.
```

Comment:
One alternative would be to try appending some characters to the file/directory name in order write something...

But that may be too complicated and it's definitely beyond the scope of this PR

Reply:
Agreed